docs/src/functions/financial/fv.md

```diff
@@ -4,8 +4,46 @@ outline: deep
 lang: en-US
 ---
 
-# FV
+# FV function
+<!--@include: ../markdown-snippets/standard-headings.md{1,1}-->
+FV (<u>F</u>uture <u>V</u>alue) is a function of the Financial category that can be used to predict the future value of an investment or asset based on its present value.
 
-::: warning
-🚧 This function is implemented but currently lacks detailed documentation. For guidance, you may refer to the equivalent functionality in [Microsoft Excel documentation](https://support.microsoft.com/en-us/office/excel-functions-by-category-5f91f4e9-7b42-46d2-9bd1-63f26a86c0eb).
-:::
\ No newline at end of file
+FV can be used to calculate future value over a specified number of compounding periods. A fixed interest rate or yield is assumed over all periods, and a fixed payment or deposit can be applied at the start or end of every period.
+
+If your interest rate varies between periods, use the [FVSCHEDULE](./FVSCHEDULE) function instead of FV.
+<!--@include: ../markdown-snippets/standard-headings.md{2,2}-->
+<!--@include: ../markdown-snippets/standard-headings.md{3,3}-->
```

Comment:
Are you sure you want to do that? IMHO this makes the Markdown much more difficult to read.
You coculd always have a template that you copy/paste every time, no?